controllers/admin/AdminTranslationsController.php

```diff
@@ -208,7 +208,7 @@ public function initMain()
         $arrayStreamContext = @stream_context_create(['http' => ['method' => 'GET', 'timeout' => 8]]);
 
         if ($langPacks = Tools::file_get_contents($this->link_lang_pack, false, $arrayStreamContext)) {
-            if ($langPacks != '' && $langPacks = json_decode($langPacks, true)) {
+            if ($langPacks = json_decode($langPacks, true)) {
```

Comment:
If I am not wrong, if `$langPacks` is equal to empty string then the behavior is different? You changed the behavior?

Reply:
The `if ($langPacks =` is already taking care of a possible empty value. You can check this example :) :
https://onlinephp.io/c/00674